qtp_diversity/__init__.py

```diff
@@ -23,12 +23,12 @@
     QiitaArtifactType('alpha_vector', 'Alpha Diversity per sample results',
                       False, False, False, [('plain_text', True),
                                             ('qza', False)]),
-    QiitaArtifactType('FeatureData[Taxonomy]', 'Taxonomy assignments', False,
+    QiitaArtifactType('FeatureData', 'Feature data', False,
```

Comment:
Can you remind me how semantic types are handled in Qiita? Is the use of the "bracket" notation purely decorative and not interpreted in any way?

Reply:
There is no interpretation; these names are really used as the storage (folder/name) location where we are going to save the files in Qiita - this give us the file path - I guess this is an interpretation?! 😄 . Anyway, when we first added this we simply added the full name. Now, it's clear that having the main name and storing all the predicates in there makes more sense.